src/displayapp/screens/settings/Settings.cpp

```diff
@@ -1,33 +1,27 @@
 #include "displayapp/screens/settings/Settings.h"
 #include <lvgl/lvgl.h>
 #include <array>
```

Comment:
`std::function` and `std::unique_ptr` are used in this file. We should include the corresponding headers [<functional>](https://en.cppreference.com/w/cpp/header/functional) and [<memory>](https://en.cppreference.com/w/cpp/header/memory)

```suggestion
#include <array>
#include <functional>
#include <memory>
```